README.md

```diff
@@ -34,4 +34,19 @@ Finally, for normal use, git clone this repository and then run:
 
 Refer to demo.ipynb for more functionality. 
 
-
+## Docker
+
+If you have [Docker](https://www.docker.com) installed, you can download this container using:
+~~~
+docker pull libsemigroups/libsemigroups-cppyy
+~~~
+
```

Comment:
Are these instructions quite right? Shouldn't we do `docker run`, looks like something went wrong here...

Reply:
Hmm yes, I must have miscopied this over! I've fixed it now.